src/lib/containers/table/SynapseTable.tsx

```diff
@@ -917,6 +947,14 @@ export default class SynapseTable extends React.Component<
             <span style={{ whiteSpace: 'nowrap' }}>Access</span>
           </div>
         </th>,
+      )      
+    }
+    if (isRowSelectionVisible) {
+      tableColumnHeaderElements.unshift(<th key="rowSelectionColumn">
+          <div className="SRC-centerContent">
+            {/* checkboxes column */}
```

Comment:
Should this have column header text 'WorkflowState'?

Reply:
No, this is the checkbox column.  The WorkflowState column is another one (that my custom command happens to be targeting).